src/howto/create-project/index.md

```diff
@@ -0,0 +1,121 @@
+
+   ![](./mergin_plugin_project_wizard_2.png)
+
+   - **New basic QGIS project**. If you are new to QGIS, this is a good starting point. The new basic QGIS project will contain a survey layer (a point layer) and a background map (OpenStreetMap).
+
+   - **Package current QGIS project**. This option will create a copy of your project and all the files in a single folder. There are three options for handling layers: package, keep as is (the layer will be referenced as is in the new project) or ignore (the layer will not be included in the new project). 
+     By default, the web services (e.g. WMTS, XYZ tiles, vector tile layers) will be referenced as they are, raster layers will be copied as they are, and vector layers will be all written to GeoPackage format (each vector in one GeoPackage database).
+
+     ![](./mergin_plugin_project_wizard_3.png)
+
+   - **Use current QGIS project as is**. This is for cases when you have already a stand-alone folder with your project packaged.
+
+4. Name the project and select a path where your project folder and associated files will be generated:
+
+   ![](./mergin_plugin_project_wizard_4.png)
+
+The new <MainPlatformName /> project will be created locally on your computer and also on the Mergin server.
+
+### Clone an existing project in QGIS
+
+Using <QGISPluginName />, you can make a copy of one of your existing projects or the ones shared with you. 
+
+1. Right-click on the project in the **Browser panel > Mergin** and select **Clone**.
+
+![](./plugin-clone.png)
+
+2. Select the **Owner** from the drop-down menu and type your new **Project Name**.
+
+![](./clone.png)
+
+## Create a project through cloudmergin.com
+
+1. Navigate to <MainDomainNameLink /> and sign in.
+
+2. To create a new project, click on **Create** in the **Projects** section:
+
+   ![](./web-create-project.png)
+
+3. Give the project a **name**
+   - in **Template project**, you can select a template from the list
+   - **Project owner** is you (your username) or your organisation
+   - the project is private by default, you can use the checkbox to make it **Public**
+
+   ![](./web-create-project-2.png)
+
+   After creating a new project, you will be directed to the project page. 
+
+4. Add files to your project:
+   - Click on **Drag & drop here to click and select file(s) to upload**
+   - Select files or folders to upload
+   - Once the files and folders are selected, press **Update**
+
+   ![Upload files](./web-project-upload.png)
+
+### Clone an existing project through cloudmergin.com
+
+You can also make a copy of an existing project or a project that is shared with you:
+
+1. Open the project link in your web browser
+2. From the top right of the panel, select **Clone**
+   ![](./web-clone-project.png)
+3. In the new window, you can change the project name and select the owner
+<!-- add screenshot? -->
+
+
+
+
+
+
```

Comment:
- Please remove extra white lines (ideally you can create a single newline at the end of the file)
- remove commented out code `<!-- add screenshot? -->`, either add screenshot, or create a ticket, I doubt anyone will remember "TODOs" in the text :)

Reply:
Cleaned up, the only 404 left is https://github.com/merginmaps/docs/edit/main/src/howto/create-project/index.md - the link from the "Help us improve this page!".

Reply:
created bug issue for this: https://github.com/MerginMaps/docs/issues/80